docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,740 @@
+8. After clicking Show Details and a successful installation, you will see the following
+
+.. image:: ../_static/control_center_pics/mac/mac_install_done.png
+
+9. Click Next and Finish to complete installation
+10. Navigate to the installation location and find the IQ Control Center folder. Inside you will find
+
+.. image:: ../_static/control_center_pics/mac/mac_inside_the_folder.png
+
+11. Double click on the IQ Control Center icon to start the application
+12. Once open you should see the following
+
+.. image:: ../_static/control_center_pics/mac/mac_control_center_open.png
+    :scale: 60%
+
+Congratulations! You have successfully installed IQ Control Center, and are ready to start communicating with your modules.
+
+**************************
+Hardware Configuration
+**************************
+
+.. _connection_guide:
+
+Connection with a Computer
+===============================
+
+In order to communicate from your computer with a Vertiq module through the Control Center, you will need some form of USB-to-UART converter.
+
+We recommend using an CP2102 FTDI such as the one found `here <https://www.amazon.com/Ximimark-Module-Serial-Converter-CP2102/dp/B07T1XR9FT?>`_.
+
+Once obtained, plug your USB-to-UART converter into your computer and install any necessary drivers (refer to the manufacturer's documentation for 
+your USB-to-UART converter). If installed correctly, a new serial port should be available on your PC. 
+Take note of the name of this new serial port. In Windows you can see this under Device Manager->Ports (COM & LPT). The image 
+below shows an example where the connected USB-to-UART converter shows up as COM3. This port will be used later when connecting with the Control Center.
+
+.. image:: ../_static/control_center_pics/windows/device_manager.png
+
+Wiring
+=============
+In order to communicate with your module(s) through IQ Control Center, you must connect serial communication wires. 
+For instructions on how to do so for your specific module, visit your module's family page:
+
+* :ref:`Vertiq 23-XX Family Serial Configuration <23xx_comms>`
+* :ref:`Vertiq 40-XX Family Serial Configuration <40xx_comms>`
+* :ref:`Vertiq 81-XX Family Serial Configuration <81xx_comms>`
+
+On modules with exposed serial pads, we recommend using cabling with one end set into a Servo JR female connector, and the other 
+end soldered directly to the module's pads. On the 81-08 G1, we recommend using a cable with a Servo JR female connector on both sides.
+
+Using Servo JR connectors makes it easy to connect with, and disconnect from the USB-to-UART device that you are using to connect with the Control Center. 
+
+Single Module Wiring
+---------------------------
+To communicate with a single module, simply connect the Servo JR female connector to the FTDI.
+
+**Please ensure that your USB-to-UART device's TX port is connected to your module's RX port, and your module's TX to the USB-to-UART device's RX.**
```

Comment:
In the first part of this, you say TX port and RX port, then in the second half its just TX and RX, no port. May want to pick one for consistency